### ISSUE TITLE
CreateWorkItem duplicate fields WIQL doesn't handle nulls

### ISSUE THREAD

**Issue by s-burtonshaw** [MicrosoftPremier/VstsExtensions#147]
The WIQL built from the duplicates fields does not handle nulls intuitively, For example. if you want to prevent a work item being raised if a similar one is still open (i.e., not closed) you might try including ```Microsoft.VSTS.Common.ClosedDate``` as a key field. This doesn't work due to what looks like some bad JS type conversion:

**Expected WIQL** 
SELECT System.Id FROM WorkItems WHERE System.TeamProject = 'Foo' AND [System.Title] = 'Bar' AND [Microsoft.VSTS.Common.ClosedDate] = ''

**Actual WIQL**
SELECT System.Id FROM WorkItems WHERE System.TeamProject = 'Foo' AND [System.Title] = 'Bar' AND [Microsoft.VSTS.Common.ClosedDate] = 'undefined'

This causes an error in the logs like:
```
2021-06-14T17:50:47.4196343Z TF212023: You cannot compare fields with different data types in the WHERE clause of a work item query. The error is caused by «[Microsoft.VSTS.Common.ClosedDate] = 'undefined'».
```

P.s We are using Azure DevOps (not on prem) with version 1.* of the extension. 

**tl;dr**
Just a bit of context on our use-case, We are trying to setup our release pipeline to raise a bug when a previous step (that runs some Cucumber BDD tests) fails _but_ only if another active bug doesn't already exist in the current sprint i.e., so if the same tests are still failing on subsequent releases deployment we don't keep raising bugs. We could potentially log exactly which tests failed in the bug (e.g., description) and check that for duplication but we don't have easy access to that info and it would complicates matters thinking about different subsets of failed tests.

**Comment by ReneSchumacher** [comment]
Hi @s-burtonshaw,

thanks for reporting this. I'll check the WIQL generation and fix the issue asap. We're adding a couple more features to the Create Work Item task, so it might take until the weekend before we actually release the new version.

**Comment by s-burtonshaw** [comment]
Any update @ReneSchumacher ?

**Comment by ReneSchumacher** [comment]
Hi @s-burtonshaw,

sorry for the delay. The review process took longer than I expected. The release for the new version is now running and it should be published to the marketplace within the hour.

**Comment by s-burtonshaw** [comment]
@ReneSchumacher No problem! That's great thanks.

**Comment by ReneSchumacher** [comment]
Let me know if you still see any issues.

**Comment by ReneSchumacher** [comment]
I'm closing this issue now as it has been resolved. Please add additional comments or open a new issue if needed.

**Comment by s-burtonshaw** [comment]
Thanks @ReneSchumacher I can confirm it looks to have fixed our issue (sorry for the delay was waiting for a failure in our pipeline!)
